plugins/outputs/health/health.go

```diff
@@ -219,10 +229,12 @@ func (h *Health) isHealthy() bool {
 
 func NewHealth() *Health {
 	return &Health{
-		ServiceAddress: defaultServiceAddress,
-		ReadTimeout:    config.Duration(defaultReadTimeout),
-		WriteTimeout:   config.Duration(defaultWriteTimeout),
-		healthy:        true,
+		ServiceAddress:        defaultServiceAddress,
+		ReadTimeout:           config.Duration(defaultReadTimeout),
+		WriteTimeout:          config.Duration(defaultWriteTimeout),
+		MaxTimeBetweenMetrics: 0,
+		healthy:               true,
+		lastMetricTime:        time.Time{},
```

Comment:
You don't need to initialize the two fields as Golang always uses the `nil` value when creating new instances. So go with
```suggestion
		healthy:               true,
```